recipe-zig/recipe.yaml

```
@@ -75,7 +75,7 @@ outputs:
 
     requirements:
       build:
-        - zig ==0.13.0
+        - zig >=0.14.0,<0.15.2
```

0.15.2 is broken on conda-forge, right? Shall we skip it instead?

```suggestion
        - zig >=0.14.0,!=0.15.2
```

---

The upstream issue in Zig has been fixed, so we can remove this constraint.

<details>
  <summary>off topic</summary>
  
  TIL: I thought it might be better to has a upper bound to prevent future failures but after reading some [arti](https://hynek.me/articles/semver-will-not-save-you/)[*cles*](https://iscinumpy.dev/post/bound-version-constraints/) I'll just leave it as is.
</details>